lib/common/actions/index.js

```diff
@@ -131,6 +131,9 @@ function getErrorMessageFromJson (
     }
     // re-assign error message after it gets used in the detail.
     errorMessage = json.message || JSON.stringify(json)
+    if (json.detail.includes('Key field must be unique')) {
```

Comment:
I believe the detail of the JSON response has changed with the new backend PR (Server PR #[489](https://github.com/ibi-group/datatools-server/pull/489))to be in the format of `"Trip id 2 conflicts with an existing trip id."`.

Reply:
This check here will need to be updated to avoid the reload button appearing. String matching also feels a little brittle but should be fine since this is a temp fix